tests/test_decoder.py

```diff
@@ -63,9 +63,9 @@ def test_end_after_filename():
         sabctools_yenc_wrapper(data_plain)
 
 
-def test_bad_size():
+def test_empty():
     with pytest.raises(ValueError) as excinfo:
-        sabctools.yenc_decode(bytearray())
+        sabctools.yenc_decode(memoryview(b""))

```

Comment:
This should not matter, just highlighting that the test has changed and isn't what we'd usually call it with.
It is now testing a memoryview of bytes instead of bytearray.

I don't think there is anything wrong with allowing either.

Reply:
True!